src/routes/LandingPage.tsx

```diff
@@ -0,0 +1,56 @@
+import { Text, Button, Box} from "@chakra-ui/react";
+
+interface LandingPageProps {
+    text: string;
+}
+
+const LandingPage: React.FC<LandingPageProps> = ({ text }) => {
+    const NavBar: React.FC = () =>
+    {
+        return(
+            <div style={{ position: 'fixed', top: 10, right: 0, padding: '10px' }}>
+                <ul style={{ listStyleType: 'none', margin: 0, padding: 10 }}>
+                    <li style={{ display: 'inline', marginRight: '10px' }}><a href="/">Speakers</a></li>
+                    <li style={{ display: 'inline', marginRight: '10px' }}><a href="/auth"> PuzleBang </a></li>
+                    <li style={{ display: 'inline', marginRight: '10px' }}><a href="/contact">Login</a></li>
+                </ul>
+            </div>
+        );
+    }
```

Comment:
Let's make this into its own component! So basically make a `components` folder, make a `NavBar.tsx` file in that folder, and then have that export a navbar component and import that here